src/ergw_aaa_diameter.erl

```diff
@@ -218,6 +218,13 @@ validate_option(Opt, Value) ->
 validate_option_error(Opt, Value) ->
     throw({error, {options, {Opt, Value}}}).
 
+decode_diameter_uri(Value) ->
+    Module = diameter_types, % trick to stop xref complains about undef function
+    try
+        apply(Module, 'DiameterURI', [decode, Value])
+    catch _:_ -> apply(Module, 'DiameterURI', [decode, Value, #{rfc => 6733}]) % OTP 20
```

Comment:
I would prefer to have the newer function tried first and the fall back to the older one.